incubator/hnc/pkg/testutils/testutils.go

```diff
@@ -332,13 +370,17 @@ func RecoverHNC() {
 		a = "recover-test-a"
 		b = "recover-test-b"
 	)
-	CleanupNamespaces(a, b)
+	// Need to explicitly call deleting these two namespaces because this function is called from the issues test
+	// ("Should allow deletion of namespaces with propagated objects that can't be removed - issue #1214", which tries
+	// to delete a specific namespace "child" . Cleaning up all namespaces using CleanupTestNamespaces will cause that
+	// test to fail.
+	cleanupNamespaces(a,b)
 	// Ensure validators work
-	mustRunWithTimeout(1, 30, "kubectl create ns", a)
+	createNamespaceWithLabels(a,"",30)
```

Comment:
Since you're deleting these namespaces by name, not label, you don't have to use the `createNamespaceWithLabels` function - you can just go back to the old way.

Reply:
Yes, will revert